src/main/java/frc/robot/subsystems/conveyor/commands/Convey.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.subsystems.conveyor.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+/**
+ * This command convey the balls to the shooter, or to the funnel.
+ */
+public class Convey extends CommandBase {
+    private final Conveyor conveyor;
+    private final double power;
+    private final boolean toShooter;
+
+    public Convey(Conveyor conveyor, double power, boolean toShooter) {
+        this.conveyor = conveyor;
+        this.power = power;
+        this.toShooter = toShooter;
+
+        addRequirements(conveyor);
+    }
+
+    @Override
+    public void execute() {
+        if (toShooter) {
+            if (Conveyor.getBallsAmount() > 0)
```

Comment:
I'm not sure this condition is desirable. You want to be able to shoot even if the sensor malfunctions.

Reply:
Can you please elaborate?

Reply:
I see you're using my words againat me 🙃
Say there's a problem in the sensor and the Power Cells counter says there are `-5` Power Cells in the robot, while in fact there are `2`. This condition won't allow shooting.

you could argue that if the sensor malfunctions, other features won't work (I believe, for example, you'll prevent intaking Power Cells if there are 5 in the robot). However, in that specific case you won't lose anything from running the conveyor, even if it's empty.

Reply:
You'll lost some power (from the battery), and if you really want, we can always add an "override" mode, that override almost, if not every check. Anyway, I don't think I really should add the checks here, (in my opinion) we need to create the "safety features" in another branch. I just did it because I knew that you would (probably) give me a comments about that I don't check some of the cases